packages/ui/src/molecules/QuantitySelector/QuantitySelector.tsx

```diff
@@ -0,0 +1,77 @@
+import type { InputHTMLAttributes } from 'react'
+import React from 'react'
+
+import type { InputProps } from '../../atoms/Input'
+import Input from '../../atoms/Input'
+import type { IconButtonProps } from '../IconButton'
+import IconButton from '../IconButton'
+
+export interface QuantitySelectorProps
+  extends InputHTMLAttributes<HTMLDivElement> {
+  /**
+   * Sets the current value that should be displayed on the input at the center of the quantity selector.
+   */
+  quantity: number | string
+
+  /**
+   * Object with properties that will be passed forward the inner IconButton atom at the left of the input.
+   */
+  leftButtonProps: IconButtonProps
+
+  /**
+   * Object with properties that will be passed forward the inner Input atom between the two buttons.
+   */
+  inputProps: InputProps
+
+  /**
+   * Object with properties that will be passed forward the inner IconButton atom at the right of the input.
+   */
+  rightButtonProps: IconButtonProps
+
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   *
+   * @default 'store-quantity-selector'
+   */
+  testId?: string
+}
+
+const QuantitySelector = ({
```

Comment:
I'd recommend sticking to our usual `forwardRef` behavior you can find in some other components.

Reply:
I agree! Missed that one out. Done at https://github.com/vtex/faststore/pull/1083/commits/baa3c56214bd32914cd60a4b2ab7d97cbd368552